src/api.js

```diff
@@ -280,6 +280,45 @@ api.declare({
 });
 
 api.declare({
+  name: 'badges',
+  title: 'Latest Build Status Badge',
+  description: [
+    'Checks the status of the latest build of a given branch ',
+    'and returns corresponding badge image.',
+  ].join('\n'),
+  stability: 'experimental',
+  method: 'get',
+  route: '/badge/:owner/:repo/:branch',
+}, async function(req, res) {
+  // Extract owner, repo and branch from request into variables
+  let {owner, repo, branch} = req.params;
+
+  // Look up the installation ID in Azure. If no such owner in the table, no error thrown
+  let ownerInfo = await this.OwnersDirectory.load({owner}, true);
+
+  if (ownerInfo) {
+    try {
+      let instGithub = await this.github.getInstallationGithub(ownerInfo.installationId);
+      // Statuses is an array of status objects, where we find the relevant status
+      let statuses = await instGithub.repos.getStatuses({owner, repo, ref: branch});
+      let status = statuses.find(statusObject => statusObject.context.startsWith(`${this.cfg.app.statusContext}`));
```

Comment:
Yeah, there's 'creator' key, and inside there is 'taskcluster[bot]' as login and the id. Sure, I can do that